test/sqlsmith/mzcompose.py

```diff
@@ -72,6 +72,8 @@
     "operator does not exist",  # For list types
     "couldn't parse role id",
     "unrecognized privilege type:",
+    "cannot return complex numbers",
```

Comment:
For my understanding: According to the comment above, these are error messages that are valid and expected due to generated statements, which are not valid? Is this correct?

Reply:
Yes. Basically SQLsmith is too dumb to instruct it that the valid parameters to functions need to have some constraints. (And maybe we don't to restrict that to check that we don't crash at least.)
So those query errors will keep happening, there is no associated issue (for those we use ci-regexp: https://github.com/MaterializeInc/materialize/blob/main/doc/developer/ci-regexp.md), and thus we just silence them.